webapp/src/js/services/enketo-translation.js

```diff
@@ -242,17 +242,29 @@ angular.module('inboxServices').service('EnketoTranslation',
       return root.xml();
     };
 
+    // DO NOT MERGE until this PR has landed: https://github.com/medic/medic/pull/5643/
```

Comment:
Should we remove this comment since it's now merged?

Reply:
This PR needs to be rebased to pick up the changes in that PR. On it.